src/assets/notificationsSuccess.svg

```diff
@@ -0,0 +1 @@
+<svg width="181" height="181" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="90.949" cy="90.926" r="90" fill="#87F1CB" fill-opacity=".2"/><g filter="url(#filter0_d)"><circle cx="90.947" cy="90.922" r="44.855" fill="url(#paint0_linear)"/></g><path d="M110.799 74.964l.106.106-.106-.106-28.436 28.395-10.865-10.85a2.535 2.535 0 00-3.581 0l.106.107-.106-.106a2.526 2.526 0 000 3.576l12.656 12.638a2.536 2.536 0 003.58 0L114.38 78.54a2.526 2.526 0 000-3.576 2.535 2.535 0 00-3.581 0z" fill="#fff" stroke="#fff" stroke-width=".3"/><defs><linearGradient id="paint0_linear" x1="60.602" y1="153.225" x2="121.602" y2="53.225" gradientUnits="userSpaceOnUse"><stop stop-color="#37B992"/><stop offset="1" stop-color="#35D6A7"/></linearGradient><filter id="filter0_d" x="43.092" y="44.067" width="95.71" height="95.71" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1.5"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter></defs></svg>
```

Comment:
Should we export `notificationsError` also as an SVG?

Reply:
will merge and ask paty for the svg version, she sent this asset because i was not able to export it from figma for some reason